src/translations/en/budget.json

```diff
@@ -0,0 +1,4 @@
+{
+  "totalExpendituresGraph.axis.left.legend": "cumulative total in €",
```

Comment:
I also saw a `'cost per month'` somewhere which I'm not sure is displayed anywhere

Reply:
That an non-displayed ID for the `TotalExpendituresGraph` component ;)